src/root/views/countries.js

```diff
@@ -363,7 +363,7 @@ class AdminArea extends SFPComponent {
             <FilterHeader
               id='dtype'
               title={strings.countryTableDisasterType}
-              options={dTypeOptions}
+              options={[{ value: 'all', label: 'All Types' }, ...this.props.disasterTypesSelect ]}
```

Comment:
@frozenhelium @batpad I had to replace these in all places (~3-4) because I couldn't implement using the `disasterTypesSelect` from the `state` in the `utils/utils.js`